include/dsn/service_api_c.h

```diff
@@ -500,7 +500,7 @@ extern DSN_API bool         dsn_semaphore_wait_timeout(
 //------------------------------------------------------------------------------
 
 // rpc address utilities
-
+extern DSN_API int           dsn_address_use_ip_as_name;
```

Comment:
This seems to be a dsn-internal configuration. I'd suggest keeping it private to the configurations.